packages/shared/forks/ReactFeatureFlags.native-fb.js

```diff
@@ -9,6 +9,11 @@
 
 import typeof * as FeatureFlagsType from 'shared/ReactFeatureFlags';
 import typeof * as ExportsType from './ReactFeatureFlags.native-fb';
+import * as DynamicFeatureFlags from './ReactFeatureFlags.www-dynamic';
```

Comment:
This should be a new file specific to RN, like './ReactFeatureFlags.native-fb-dynamic'.

Then you need to setup Jest to route to the correct file, like we do for www: https://github.com/facebook/react/blob/f3337aa544998be249c63ad3accc5008dbb938ed/scripts/jest/setupTests.www.js#L4-L8

The trouble is that we don't have an RN specific test run. Could you move all our RN tests into their own Jest project, like we do for DevTools? That's the direction we're headed, where each renderer has its own project. Like how the `yarn flow` command works.

Reply:
Ah right - I'm just going to hardcode the setting to false for now then to unblock and come back to this while working out how this flag works in www